src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -172,6 +174,7 @@
  *   -R reference.fasta \
  *   -V input.vcf \
  *   -O output.vcf \
```

Comment:
You sure the output file should still have a name of ".vcf" if the output format is MAF?

Reply:
Good point.  Fixed!